scripts/prepare-release.sh

```diff
@@ -2,30 +2,50 @@
 set -e
 set -o pipefail
 
-PREVIOUS_VERSION=$(grep integreatly-operator deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | awk -F v '{print $2}')
+if [[ -z "$TYPE" ]]; then
+  TYPE="integreatly-operator"
+fi
+
+PREVIOUS_VERSION=$(grep $TYPE deploy/olm-catalog/$TYPE/$TYPE.package.yaml | awk -F v '{print $2}') || echo "No previous version"
+
+if [[ -z "$ORG" ]]; then
+  ORG="integreatly"
+else
+  ORG="$ORG"
+fi
 
 create_new_csv() {
-  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name integreatly-operator --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+  if [[ -z "$PREVIOUS_VERSION" ]]
+    then
+      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$TYPE" --csv-channel=rhmi --update-crds
+    else
+      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$TYPE" --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+  fi
 }
 
 update_csv() {
-  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name integreatly-operator --csv-channel=rhmi --update-crds
+  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$TYPE" --csv-channel=rhmi --update-crds
 }
 
 set_version() {
-  "${SED_INLINE[@]}" "s/$PREVIOUS_VERSION/$VERSION/g" Makefile
-  "${SED_INLINE[@]}" "s/$PREVIOUS_VERSION/$VERSION/g" version/version.go
+  if [[ -z "$PREVIOUS_VERSION" ]]
```

Comment:
Looks like we need `$PREVIOUS_VERSION` to be set earlier in the script too. Wdyt about having that as an earlier check and not continuing unless it's set?

Reply:
If we set this check to exit the script earlier it would remove the possibility of creating the initial CSV files. These initial CSV file would be require been created in a manual process.